lib/spack/docs/packaging_guide.rst

```diff
@@ -4379,10 +4379,17 @@ implementation was selected for this build:
    elif "mvapich" in spec:
        configure_args.append("--with-mvapich")
 
-It's also a bit more concise than satisfies.  The difference between
-the two functions is that ``satisfies()`` tests whether spec
-constraints overlap at all, while ``in`` tests whether a spec or any
-of its dependencies satisfy the provided spec.
+It's also a bit more concise than satisfies.
+
+.. note::
+
+   The ``satisfies()`` method tests whether spec constraints intersect, while
+   ``in`` tests whether a spec or any of its dependencies satisfy the provided
+   spec.
```

Comment:
```suggestion
   The ``satisfies()`` method tests whether this spec has, at least, all the constraints of the argument spec,
   while ``in`` tests whether a spec or any of its dependencies satisfy the provided spec.
```

Reply:
My bad.  I was trying to minimize differences for what I'd thought was a "quick" change.

Reply:
No worries!